core/src/components/button/button.ios.vars.scss

```
@@ -108,7 +108,7 @@ $button-ios-small-height:                             2.1em !default;
 $button-ios-small-border-radius:                      6px !default;
 
 /// @prop - Font size of the small button
-$button-ios-small-font-size:                          13px !default;
+$button-ios-small-font-size:                          0.8125rem !default;
```

```suggestion
$button-ios-small-font-size:                          min(0.8125rem, 19px) !default;
```

Should I set the max size here to 4px less than clear?